addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/report/ExtensionAuthhelperReport.java

```diff
@@ -174,22 +179,41 @@ public void handle(ReportData reportData) {
                          * The AUTH_SUCCESS_STATS / AUTH_FAILURE_STATS stats can get raised on another domain.
                          * Any successes are good, but just failures are bad.
                          */
-                        addSummaryItem(
-                                ard,
-                                "auth",
+                        boolean hasSuccessStats =
+                                (inMemoryStats.getStat(
+                                                hostname, AuthenticationHelper.AUTH_SUCCESS_STATS)
+                                        != null);
+                        boolean notHasFailureStats =
```

Comment:
`notHasFailureStays` is a bit awkward. Why not use `hadFailureStats` (as per hasSuccessStats) and then just reverse the conditionals, iei ` ! hasFailureStats`

Reply:
Ya, I'm good with that, you're right the double negative is horrible

Reply:
Adjusted